src/js/core/range.js

```diff
@@ -488,12 +483,12 @@ define([
        * insert node at current cursor
        *
        * @param {Node} node
-       * @param {Boolean} [isInline]
        * @return {Node}
        */
-      this.insertNode = function (node, isInline) {
+      this.insertNode = function (node) {
```

Comment:
@easylogic for simple interface. I remove second parameter and test inside whether first argument is inline node or not.